### PR TITLE
Style: Formatting and cleaning up doc with grammatical corrections

### PR DESCRIPTION
**Pull request by Harsha509**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

### Description
As users started browsing the documentation, they were reporting grammatical issues recently. This PR is used to update doc without grammatical errors.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/introduction/on_test_automation.ko.md [hunk 1]
```diff
@@ -392,3 +392,17 @@ and pre-configure a unicorn via the API or database.
...
+
+It is not always advantageous to automate test cases. There are times when
+manual testing may be more appropriate. For instance, if the application’s user
+interface will change considerably in the near future, then any automation
+might need to be rewritten anyway. Also, sometimes there simply is not enough
+time to build test automation. For the short term, manual testing may be more
+effective. If an application has a very tight deadline, there is currently no
+test automation available, and it’s imperative that the testing gets done within
+that time frame, then manual testing is the best solution.
+
```
Comment (Harsha509) [inline docs_source_files/content/introduction/on_test_automation.ko.md]: The entire paragraph was missed out in on_test_automation.ko.md page

**Comment by Harsha509** [comment]
This PR has document cleaned up for **getting started** and **introduction** section. Moving PR for ready to review to keep PR small and easy for review.

File: docs_source_files/content/introduction/on_test_automation.de.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ weight: 2
 auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
 zu übersetzen indem Du uns einen Pull Reqeust schickst!
  {{% /notice %}}
-First, start by asking yourself whether or not you really need to use a browser.
+First, start by asking yourself whether or not you need to use a browser.
 Odds are that, at some point, if you are working on a complex web application,
-you will need to open a browser and actually test it.
+you will need to open a browser and test it.
```
Comment (diemol) [inline docs_source_files/content/introduction/on_test_automation.de.md]: I think the purpose of those words was to set a tone where the reader should reflect if they need WebDriver or not. Let's leave it like it is.

File: docs_source_files/content/introduction/on_test_automation.de.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ you can have many tests with minimal flake.
 
 A distinct advantage of Selenium tests
 are their inherent ability to test all components of the application,
-from backend to frontend, from a user's perspective.
+from the backend to frontend, from a user's perspective.
```
Comment (diemol) [inline docs_source_files/content/introduction/on_test_automation.de.md]: I believe this is not needed.

File: docs_source_files/content/introduction/on_test_automation.de.md [hunk 4]
```diff
@@ -323,7 +323,7 @@ unicornConfirmationPage = addUnicornPage.createUnicorn(sparkles)
 {{< / code-tab >}}
 
 Now that you have configured your unicorn,
-you need to move on to step 3: making sure it actually worked.
+you need to move on to step 3: making sure it worked.
```
Comment (diemol) [inline docs_source_files/content/introduction/on_test_automation.de.md]: Same as the first comment in the review

File: docs_source_files/content/introduction/on_test_automation.de.md [hunk 5]
```diff
@@ -370,13 +370,13 @@ even if Larry decides next week that he no longer likes Ruby-on-Rails
 and decides to re-implement the entire site
 in the latest Haskell bindings with a Fortran front-end.
 
-Your page objects will require some small maintenance
-in order to conform to the site redesign,
```
Comment (diemol) [inline docs_source_files/content/introduction/on_test_automation.de.md]: I believe the paragraph was already correct, what is the motivation to edit it?

File: docs_source_files/content/introduction/on_test_automation.de.md [hunk 6]
```diff
@@ -370,13 +370,13 @@ even if Larry decides next week that he no longer likes Ruby-on-Rails
...
+Your page objects will require some small maintenance 
+to conform to the site redesign,
 but these tests will remain the same.
 Taking this basic design,
 you will want to keep going through your workflows with the fewest browser-facing steps possible.
 Your next workflow will involve adding a unicorn to the shopping cart.
-You will probably want many iterations of this test in order to make sure the cart is keeping its state properly:
```
Comment (diemol) [inline docs_source_files/content/introduction/on_test_automation.de.md]: I believe the paragraph was already correct, what is the motivation to edit it?

File: docs_source_files/content/introduction/on_test_automation.en.md [hunk 7]
```diff
@@ -3,9 +3,9 @@ title: "On test automation"
 weight: 2
 ---
 
-First, start by asking yourself whether or not you really need to use a browser.
```
Comment (diemol) [inline docs_source_files/content/introduction/on_test_automation.en.md]: Same comments as in the DE version file.

File: docs_source_files/content/introduction/on_test_automation.de.md (hunk 2 above)
Comment (Harsha509) [reply docs_source_files/content/introduction/on_test_automation.de.md]: Reverted!

File: docs_source_files/content/introduction/on_test_automation.de.md (hunk 3 above)
Comment (Harsha509) [reply docs_source_files/content/introduction/on_test_automation.de.md]: Removed.

File: docs_source_files/content/introduction/on_test_automation.de.md (hunk 4 above)
Comment (Harsha509) [reply docs_source_files/content/introduction/on_test_automation.de.md]: Reverted.

File: docs_source_files/content/introduction/on_test_automation.de.md (hunk 6 above)
Comment (Harsha509) [reply docs_source_files/content/introduction/on_test_automation.de.md]: To maintain the conciseness I have replaced "in order to" with 'to'

Reverted!

File: docs_source_files/content/introduction/on_test_automation.de.md (hunk 5 above)
Comment (Harsha509) [reply docs_source_files/content/introduction/on_test_automation.de.md]: To maintain the conciseness I have replaced "in order to" with 'to'.

Reverted.

File: docs_source_files/content/introduction/on_test_automation.en.md (hunk 7 above)
Comment (Harsha509) [reply docs_source_files/content/introduction/on_test_automation.en.md]: Reverted for other pages having these changes.

**Pull request merged**